lib/src/progress_bar.dart

```diff
@@ -0,0 +1,206 @@
+import 'package:chewie/src/chewie_progress_colors.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:video_player/video_player.dart';
+
+class VideoProgressBar extends StatefulWidget {
+  VideoProgressBar(
+    this.controller, {
+    ChewieProgressColors? colors,
+    this.onDragEnd,
+    this.onDragStart,
+    this.onDragUpdate,
+    Key? key,
+    required this.barHeight,
+    required this.handleHeight,
```

Comment:
I only realized afterwards that `handleHeight` is `6` for both Material and Cupertino, but I then decided to leave it in.

Reply:
That's fine!